cmd/deploy/validate.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2022 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package deploy
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+func validateOptionVars(variables []string) error {
```

Comment:
I don't think we should call this validate, this is validating AND setting the environment variables for the command. Can we do that in two separate functions, to make it easy to read?